thirdparty/runtime_environments_test.go

```diff
@@ -21,3 +22,22 @@ func TestGetImageNames(t *testing.T) {
 	assert.NotEmpty(result)
 	assert.NotContains(result, "")
 }
+
+func TestGetPackages(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	assert := assert.New(t)
+	config := testutil.TestConfig()
+	testutil.ConfigureIntegrationTest(t, config, "TestGetPackages")
+	c := NewRuntimeEnvironmentsClient(config.RuntimeEnvironments.BaseURL, config.RuntimeEnvironments.APIKey)
+	opts := PackageFilterOptions{
+		Page:    0,
+		Limit:   10,
+		Manager: "pip",
+	}
+	result, err := c.getPackages(ctx, "ami-0e12ef25a5f7712a4", opts)
+	require.NoError(t, err)
+	assert.NotEmpty(result)
+	assert.Len(result, 10)
```

Comment:
Can you test some of the output here? And can we add a few test cases to this to verify that the options are respected?

Reply:
The test I just added checks that the manager field and name field are respected. Let me know if I should do more extensive testing.